package.json

```diff
@@ -18,6 +18,7 @@
   "license": "MIT",
   "devDependencies": {
     "coffee-script": "^1.9.1",
+    "docha": "git://github.com/thompsnm/docha.git",
```

Comment:
Gotcha. The author of docha is considering a [pull request](https://github.com/tehsenaus/docha/pull/1) from me to add the functionality I need to the main repository. If that gets merged and published to npm soon I'll change this to the default format of just specifying a version. If not, I'll modify this to use my commit as you suggest.